src/test/java/org/jabref/logic/layout/LayoutEntryTest.java

```diff
@@ -70,29 +70,29 @@ public String layout(String layoutFile, BibEntry entry) throws IOException {
+
+        assertEquals(2, LayoutEntry.parseMethodsCalls("bla,foo").size(), "Parsing of layout entry method calls failed. Not all arguments were parsed correctly.");
+        assertEquals("bla", (LayoutEntry.parseMethodsCalls("bla,foo").get(0)).get(0), "Parsing of layout entry method calls failed. Input did not match expected output. Method calls not separated as expected.");
+        assertEquals("foo", (LayoutEntry.parseMethodsCalls("bla,foo").get(1)).get(0), "Parsing of layout entry method calls failed. Input did not match expected output. Method calls not separated as expected.");
+
+        assertEquals(2, LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").size(), "Parsing of layout entry method calls failed. Check for escaped characters behaviour.");
+        assertEquals("bla", (LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").get(0)).get(0), "Parsing of layout entry method calls failed. Input did not match expected output. Check for multiple method calls with escaped characters.");
+        assertEquals("foo", (LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").get(1)).get(0), "Parsing of layout entry method calls failed. Input did not match expected output. Check for multiple method calls with escaped characters.");
+        assertEquals("test", (LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").get(0)).get(1), "Parsing of layout entry method calls failed. Input did not match expected output. Check for multiple method calls with escaped characters.");
+        assertEquals("fark", (LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").get(1)).get(1), "Parsing of layout entry method calls failed. Input did not match expected output. Check for multiple method calls with escaped characters.");
+
+        assertEquals(2, LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").size(), "Parsing of layout method calls failed. Check for arguments in parentheses.");
+        assertEquals("bla", (LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").get(0)).get(0), "Parsing of layout method calls failed. Input did not match expected output. Check for arguments in parentheses.");
+        assertEquals("foo", (LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").get(1)).get(0), "Parsing of layout method calls failed. Input did not match expected output. Check for arguments in parentheses");
+        assertEquals("test", (LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").get(0)).get(1), "Parsing of layout method calls failed. Input did not match expected output. Check for arguments in parentheses");
+        assertEquals("fark", (LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").get(1)).get(1), "Parsing of layout method calls failed. Input did not match expected output. Check for arguments in parentheses");
```

Comment:
It seems to me that the first sentence "Parsing of layout entry method calls failed." is superfluous, since this is what the method name implies (which is displayed when assert fails). Maybe just remove the first sentence and change the method name to something more meaningful?

Reply:
Much work todo, I just reverted the change.